velox/docs/monthly-updates/january-2023.rst

```diff
@@ -0,0 +1,82 @@
+********************
+January 2023 Update
+********************
+
+Core Library
+============
+
+* Support for Generic writer.
```

Comment:
`Generic -> a generic`
Can you add a couple more words about the generic writer?